src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/converter/StringListConverter.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.converter
+
+import javax.persistence.AttributeConverter
+import javax.persistence.Converter
+
+@Converter
+class StringListConverter : AttributeConverter<List<String?>?, String?> {
+  companion object {
+    private const val SPLIT_CHAR = ";"
+  }
+
+  override fun convertToDatabaseColumn(stringList: List<String?>?): String? {
+    return stringList?.joinToString(separator = SPLIT_CHAR)
+  }
```

Comment:
when we come to test whether today's list of teams is the same as yesterdays, how are you planning to do that comparison? Would it be helpful to have the list in an ordered form?

Reply:
that took me a minute to figure out (on an other PR). Converting to Set (unordered) and doing an equals comparison seems to work